### PR TITLE
[TM-13] Create Pledge Pages

### PR DESCRIPTION
**Pull request by mmiddleton3301**
Contains the 'start' page for creating a pledge and links back to EAS.

Also see:

* https://github.com/SkillsFundingAgency/das-employerapprenticeshipsservice/pull/2181
* https://github.com/SkillsFundingAgency/das-employer-config/pull/871

### CODE REVIEW

File: src/SFA.DAS.LevyTransferMatching.Web.UnitTests/Controllers/PledgesControllerTests.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+using NUnit.Framework;
+using SFA.DAS.LevyTransferMatching.Web.Controllers;
+using SFA.DAS.LevyTransferMatching.Web.Models.Pledges;
+
+namespace SFA.DAS.LevyTransferMatching.Web.UnitTests.Controllers
+{
+    [TestFixture]
+    public class PledgesControllerTests
+    {
+        private PledgesController pledgesController;
```
Comment (chrisfoster76) [inline src/SFA.DAS.LevyTransferMatching.Web.UnitTests/Controllers/PledgesControllerTests.cs:11]: Could we prefix this with `_` to denote a private field?

File: src/SFA.DAS.LevyTransferMatching.Web.UnitTests/Controllers/PledgesControllerTests.cs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        public void Setup()
+        {
+            pledgesController = new PledgesController();
+        }
+
+        [Test]
+        public void GET_Index_Returns_Expected_View_With_Expected_ViewModel()
+        {
+            // Arrange
+            string encodedAccountId = "abc";
```
Comment (chrisfoster76) [inline src/SFA.DAS.LevyTransferMatching.Web.UnitTests/Controllers/PledgesControllerTests.cs:23]: What about adding in the AutoFixture package and get it to create a random value for us?

File: src/SFA.DAS.LevyTransferMatching.Web.UnitTests/Controllers/PledgesControllerTests.cs [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            ViewResult viewResult = null;
```
Comment (chrisfoster76) [inline src/SFA.DAS.LevyTransferMatching.Web.UnitTests/Controllers/PledgesControllerTests.cs:24]: we could remove this line and add the declaration in the act lines, same goes for the following line

**Review by chrisfoster76 — APPROVED** [review]
Just some minor things

File: src/SFA.DAS.LevyTransferMatching.Web.UnitTests/Controllers/PledgesControllerTests.cs (hunk 1 above)
Comment (mmiddleton3301) [reply src/SFA.DAS.LevyTransferMatching.Web.UnitTests/Controllers/PledgesControllerTests.cs:11]: This was just down to the style I was working with in a previous project. Fixed.

**Pull request merged**